osf/utils/migrations.py

```diff
@@ -390,6 +390,39 @@ def create_schema_blocks_for_question(state, rs, question, sub=False):
         split_options_into_blocks(state, rs, question, schema_block_group_key)
 
 
+
+def create_schema_blocks_for_simple_schema(schema):
```

Comment:
Honestly, I wouldn't call this "simple schema". It's easier to parse but harder to read in some ways. "Explicit schema" feels more accurate.

Reply:
I changed it to atomic schema